_sass/_base.scss

```diff
@@ -29,21 +29,16 @@ p {
 }
 
 .wrapper {
-    background:$theme-color;
```

Comment:
Why remove this?

Reply:
If main is shorter you don't want it to be coloured this way at the bottom. This line doesn't add any value or make sense conceptually if flex is used. There are essentially two panels each with its own colour so there is no reason why their wrapper should also have a colour.

Reply:
@nelsonmestevao should we merge this?

Reply:
@sharu725 It adds a new "bug". So I think we shouldn't merge yet.

Reply:
Let me show it with an print.

Reply:
![2018-11-16-105235_254x361_scrot](https://user-images.githubusercontent.com/19409687/48617238-c9cb3b00-e98d-11e8-921f-270262ab9b8c.png)

Reply:
Oh, that's ugly. Thanks @nelsonmestevao.  @rudolfovic this issue happens on resizing the window I guess. Please see if it can be fixed.

Reply:
![2018-11-16-105633_1046x822_scrot](https://user-images.githubusercontent.com/19409687/48617405-4e1dbe00-e98e-11e8-8fd9-8d1a3f40ce90.png)
If the content is smaller than the sidebar, this starts to happen.